docs/common.rst

```diff
@@ -28,60 +28,62 @@ The UTF-8 encoded files MUST NOT include a BYTE ORDER MARK (U+FEFF).
 
 Line Endings
 ````````````
- * Lines in an exchange text file SHOULD end with a LINE FEED (U+000A).
- * Lines MAY also end with a CARRIAGE RETURN (U+000D) followed immediatly by a LINE FEED (U+00A).
+ * Lines in an exchange text file SHOULD end with a LINE FEED (LF) (\\n) (U+000A).
+ * Lines MAY also end with a CARRIAGE RETURN (CR) (\\r) (U+000D) followed immediatly by a LINE FEED (LF) (\\n) (U+000A).
```

Comment:
We should probably just change this to use the `:unicode_info:` directive (it's custom).